src/main/java/com/bis/instagrarn/ProfileController.java

```diff
@@ -0,0 +1,67 @@
+package com.bis.instagrarn;
+
+import java.io.File;
+import java.io.IOException;
+import java.text.DateFormat;
+import java.util.Date;
+import java.util.Locale;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.multipart.MultipartFile;
+
+import common.Common;
+
+/**
+ * Handles requests for the application home page.
+ */
+@Controller
+public class ProfileController {
+
+	@RequestMapping(value = "/addpost", method = RequestMethod.GET)
+	public String home() {
+
+		return Common.Profile.VIEW_PATH + "addpost.jsp";
+	}
+	
+	@RequestMapping(value = "/upload", method = RequestMethod.POST)
+	public String main(MultipartFile file, String content, String area, Model model) {
```

Comment:
It is also possible to create and implement boardVO :)

Reply:
okay,,,